.github/workflows/build.yml

```diff
@@ -83,7 +83,7 @@ jobs:
           python ./checker.py test
       - name: Import GPG signing key
         if: matrix.java == 16 && github.ref == 'refs/heads/main'
-        uses: crazy-max/ghaction-import-gpg@v3
+        uses: crazy-max/ghaction-import-gpg@v4
         with:
           gpg-private-key: ${{ secrets.GPG_SIGNING_KEY }}
```

Comment:
@sideshowbarker you might need to rename this and any other related changes :)

https://github.com/crazy-max/ghaction-import-gpg/releases/tag/v4.0.0

Reply:
> @sideshowbarker you might need to rename this and any other related changes :)
> 
> [crazy-max/ghaction-import-gpg@`v4.0.0` (release)](https://github.com/crazy-max/ghaction-import-gpg/releases/tag/v4.0.0)

Thanks for the heads-up — I just now updated the field names